common/persistence/elasticsearch/esVisibilityStore.go

```diff
@@ -92,7 +92,7 @@ func (v *esVisibilityStore) RecordWorkflowExecutionStarted(
 	request *p.InternalRecordWorkflowExecutionStartedRequest,
 ) error {
 	v.checkProducer()
-	msg := getVisibilityMessage(
+	msg := createVisibilityMessage(
```

Comment:
Seems like you can delete the existing methods getVisibilityMessage and getVisibilityMessageForCloseExecution